AzureDevOps/basicexample/azure-pipelines.yml

```diff
@@ -0,0 +1,44 @@
+
+# Build and test ASP.NET Core projects targeting .NET Core.
+# Add steps that run tests, create a NuGet package, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/dotnet-core
+
+trigger:
+- master
+
+pool:
+  vmImage: windows-latest
+
+variables:
+  buildConfiguration: 'Debug'
+
+
+steps:
+- task: UseDotNet@2
+  displayName: 'Use .NET Core 3.1 sdk'
+  inputs:
+    packageType: sdk
+    version: '3.1.x'
+
+- task: UseDotNet@2
+  displayName: 'Use .NET 5 sdk'
+  inputs:
+    packageType: sdk
+    version: '5.0.x'
+
+- task: UseDotNet@2
+  displayName: 'Use .NET 6 sdk'
+  inputs:
+    packageType: sdk
+    version: '6.0.x'
+
+
+- task: MicrosoftSecurityDevOps@1
+  inputs:
+    categories: all
+    policy: microsoft
+    break: true
+    languages: 'all'
+
+- script: dotnet build --configuration $(buildConfiguration)
```

Comment:
Maybe to move MicrosoftSecurityDevOps@1 after the `dotnet build` command? It will run binskim on produced binaries